x/valset/keeper/keeper_test.go

```diff
@@ -3,7 +3,8 @@ package keeper
 import (
```

Comment:
Add test cases for the new error logic flows.

Reply:
Added a error logic .Can you look into it